en/ros/external_position_estimation.md

```diff
@@ -119,17 +225,19 @@ At this point, if you followed those steps, you are ready to test your setup.
 
 Be sure to perform the following checks:
 
-* **Before** creating the rigid body, align the robot with world x axis
-* Stream over MAVLink and check the MAVLink inspector with QGroundControl, the local pose topic should be in NED
-* Move the robot around by hand and see if the estimated local position is consistent (always in NED)
-* Rotate the robot on the vertical axis and check the yaw with the MAVLink inspector
+* **Before** creating the rigid body, align the robot with world x axis.
```

Comment:
@TSC21 HOw do you alight the robot with world x axis? What does this mean?

Reply:
The world axes should be defined on the MoCap system setup. I never configured one, so I can't give an input regarding that. This just assures that the position where you start is with heading 0.